.gitlab/docker_common/publish_job_templates.yml

```diff
@@ -6,12 +6,13 @@
 
 .docker_publish_job_definition:
   image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-buildimages/deb_x64$DATADOG_AGENT_BUILDIMAGES_SUFFIX:$DATADOG_AGENT_BUILDIMAGES
-  tags: ["runner:main"]
+  tags: ["arch:amd64"]
   variables:
     <<: *docker_variables
     IMG_VARIABLES: ""
     IMG_SIGNING: ""
   script: # We can't use the 'trigger' keyword on manual jobs, otherwise they can't be run if the pipeline fails and is retried
+    - source /root/.bashrc
```

Comment:
Anything magic in there? I don't think we have it on other jobs

Reply:
Basically the same reason as in https://github.com/DataDog/datadog-agent/pull/17179: `source /root/.bashrc` is run in the entrypoint of the image, but in k8s runners, entrypoints are overwritten, so we need to run this manually.

In the case of this job, `source /root/.bashrc` runs `conda activate ddpy3`, which is necessary to get the `aws` CLI in the `PATH`.

In the long run, we want to get rid of this line, but there's some more work to do in the build images to get to this.